scylla/src/transport/session.rs

```diff
@@ -711,10 +711,7 @@ impl Session {
         let query: Query = query.into();
         let serialized_values = values.serialized()?;
 
-        let retry_session = match &query.config.retry_policy {
-            Some(policy) => policy.new_session(),
-            None => self.retry_policy.new_session(),
-        };
+        let retry_session = self.retry_policy.new_session();
```

Comment:
> As one can set policies both on session and on statement level by using
> Execution Profiles, setting policies on statements explicitly is no
> longer needed.

The commit message is confusing. `StatementConfig` doesn't hold an execution profile in this commit yet. I am cofused (at least at this point in the review, maybe it will become clear later) why you are considering the `Session`'s retry policy but not query's/statement's.